source/scripting_v3/GTA/Entities/Peds/RelationshipGroupHash.cs

```diff
@@ -0,0 +1,138 @@
+//
+// Copyright (C) 2015 crosire & contributors
+// License: https://github.com/crosire/scripthookvdotnet#license
+//
+
+using System;
```

Comment:
Don't technically need that `using` statement here, since nothing in this file references something from the `System` namespace.